tests/impl.cpp

```diff
@@ -3450,7 +3450,19 @@ result_t test_mm_extract_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_insert_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int64_t *_a = (const int64_t *) impl.mTestIntPointer1;
+    const int imm8 = 1;
+    const int64_t insert = 0xBBBBBBBBBBBBBBBB;
```

Comment:
You can use `impl.mTestIntPointer2` to assign it a value.

Reply:
Replace the value.

Reply:
It is macro. A variable is not allowed

Reply:
I do not see any macro here.
Can't it be used like
```c
const int64_t *insert = (const int64_t *) impl.mTestIntPointer2;
```
and use `*insert` in the following code?

Reply:
my fault